scss/footer.scss

```diff
@@ -0,0 +1,27 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
this is not needed, you can leave the default.

Reply:
You mean the flex-direction or the height?

Reply:
the height

Reply:
Thanks, now I remove it!